xopt/bayesian/utils.py

```diff
@@ -123,7 +123,7 @@ def check_training_data_shape(train_x: [Tensor],
                 f'does not match number of vocs {vocs_type} == ' \
                 f'{len(vocs[vocs_type])}'
         else:
-            if vocs[vocs_type] is not None:
+            if vocs[vocs_type]:
```

Comment:
Sorry, I said I had no comments, but is this right? It won't trigger if vocs[vocs_type] is not None

Reply:
I think this was my mistake, should just be if it IS none

Reply:
Can you fix?

Reply:
I think this should be: `if not vocs[vocs_type]:`. Then it will catch `{}` and `None`

Reply:
Ok thats fine, can you commit the change

Reply:
I'm sorry, it was right to begin with... I should have trusted the tests